CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+* @lokalise/frontend-foundations
```

Comment:
I wonder, when people use this template, will this file be cloned too? If they are lazy to change it, would it spam us in notifications?

Reply:
Good point, added line to README